config/webpack/server.js

```diff
@@ -48,10 +48,10 @@ var config = {
       },
       {
         test: /\.css$/,
-        include: /src\/app/,
         loaders: [
           'isomorphic-style-loader',
-          'css?modules&importLoaders=2&sourceMap&localIdentName=[local]___[hash:base64:5]'
+          'css-loader?modules&importLoaders=2&sourceMap&localIdentName=[local]___[hash:base64:5]',
+          'postcss'
```

Comment:
since we already load the bundled and preprocessed css at the server's entry point, we don't need any preprocessors